README.md

```diff
@@ -0,0 +1,36 @@
+Pimcore Bootstrap
+=================
+
+Provides Bootstrap-specific areas for your themes.
+
+Installation
+------------
+
+Using Composer:  
+
+```
+composer require pfaffenbauer/bootstrap
```

Comment:
Package was moved under pimcore-extensions community roof. Please change to "pimcore-extensions/bootstrap"